Components/lists.css

```diff
@@ -0,0 +1,59 @@
+/* Lists */
+
+.list-style-none{
+    list-style: none;
+    padding-inline-start: 0;
+}
+.inline-list{
+    display: flex;
+}
+.list-inline-item:not(:last-of-type){
+    margin-right: 1.5rem;
+}
+.styled-list{
+    margin-bottom: 1rem;
+}
+.styled-list li:not(:last-child){
+    padding-bottom: 0.625rem;
+}
+.list-style-circle{
+    list-style-type: circle;
+}
+.list-style-square{
+    list-style-type: square;
+}
+.list-style-smAlpha{
+    list-style-type:lower-alpha;
+}
+.list-style-capAlpha{
+    list-style-type:upper-alpha;
+}
+.list-style-smRoman{
+    list-style-type:lower-roman;
+}
+.list-style-capRoman{
+    list-style-type:upper-roman;
+}
```

Comment:
class naming can be improved.

Reply:
will update it to more descriptive names